.github/workflows/build.yml

```
@@ -254,14 +256,46 @@ jobs:
           echo "${{ steps.changelog.outputs.changelog }}" > ./CHANGELOG.md
           sed -i "1s/.*/## Changes since ${{ steps.last-release.outputs.release }}/" ./CHANGELOG.md
 
-      - name: Create the release
-        uses: actions/create-release@v1
+      # automatic helm repo management
+      # (from https://github.com/liqotech/liqo/blob/master/.github/workflows/integration.yml)
+      - name: Install Helm
+        uses: azure/setup-helm@v1
```

```suggestion
        uses: azure/setup-helm@v3
```

Otherwise dependabot will complain...